Parsers/CiscoMeraki/MerakiConfigurationChanges.txt

```diff
@@ -0,0 +1,20 @@
+// Title:           Cisco Meraki ConfigurationChanges API
+// Author:          Netrix LLC - Rich Lilly
+// Version:         1.0
+// Last Updated:    1/18/2021
+// Comment:         Initial Release
+//  
+// DESCRIPTION:
+// This parser takes raw Cisco Meraki logs from the ConfigurationChanges Cloud API and parses the logs into a normalized schema.
+//
+// USAGE:
+// 1. Open Log Analytics/Azure Sentinel Logs blade. Copy the query below and paste into the Logs query window. 
+// 2. Click the Save button above the query. A pane will appear on the right, select "as Function" from the drop down. Enter a Function Name.
+//    It is recommended to name the Function Alias, as CiscoMerakiConfigurationChanges
+// 3. Kusto Functions can typically take up to 15 minutes to activate. You can then use Function Alias for other queries.
+//
+// REFERENCES: 
+// Using functions in Azure monitor log queries: https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+// 
+MerakiConfigurationChanges_CL
+| project Timestamp=ts_t,adminName=adminName_s,adminEmail=adminEmail_s,adminId=adminId_s,page=page_s,label=label_s,oldValue=oldValue_s,newValue=newValue_s,networkName=networkName_s,networkId=networkId_s,ssidName=ssidName_s, ssidNumber=ssidNumber_d
```

Comment:
You are not projecting TimeGenerated here, but your sample data implies you have this value for ConfigChanges, please include that.